app/javascript/packs/application.js

```diff
@@ -328,3 +326,29 @@ $( document ).on('turbolinks:load', function() {
     }
   })
 })
+
+// This will disable the export all button when there are more than 12K records
+$( document ).on('turbolinks:load', function() {
+  // Get count of records
+  var table = $('.is-datatable').DataTable();
+  table.clear()
+  table.ajax.url($('.is-datatable').data('source')).load(function() {
+
+    var info = $('#child-objects-datatable_info').text().split(' ')
```

Comment:
Nope, all of the datatables have the 'is-datatable' class.  For example [here](https://github.com/yalelibrary/yul-dc-management/blob/main/app/views/parent_objects/index.html.erb#L25) on the parent object.  We should be safe with this identifier.

Reply:
I realize now you were actually referring to the 'child-objects-datatable_info' identifier.  This will need a refactor.  Thank you for catching this.

Reply:
I tried putting in an event listener to get the details, and it seems like it's working and updates properly when filtering has changed.
If you could use that instead of parsing the display it may work better, and will cover all tables.
(fnRecordsDisplay is the total with filters applied.)

```
dataTable = $('.is-datatable').dataTable({
 ...
      "fnDrawCallback": function( oSettings ) {
        console.log(oSettings.fnRecordsTotal() + "  -  " + oSettings.fnRecordsDisplay());
      }
 ...
 }
```